src/v2/api/controllers/AuthController.ts

```diff
@@ -152,6 +164,10 @@ export class AuthController {
     return this.authService.updatePassword(body, req.user);
   }
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: AuthLoginResponse,
```

Comment:
This response body is incorrect. Create a new response file an describe the response body with this example 
![image](https://github.com/fictadvisor/fictadvisor-api/assets/113940774/d1750994-7874-4e86-82cb-dc9a435ad8ec)

Reply:
No need to do this, OrdinaryStudentResponse already describes all fields

Reply:
Done